about.md

```diff
@@ -5,4 +5,23 @@ permalink: /about/
 nav_order: 99
 ---
 
-Add some info here about us and the project?
+# About Scrum Kata
+Scrum Kata is a hobby project by professional Scrum Masters [Henadz Kavaliou](https://github.com/henadzkavaliou) and [Benji Huser](https://github.com/huserben).
+While preparing for the PSM-III assessment we tried to answer questions around Scrum in a practice group and discussed them together afterwards.
+In the complex domain there is rarely a single answer for a problem, and in the discussions we had we learned about the different angles all those potential problems could be tackled.
+
+We also realized that we came up with many questions that we'd like to share with the community so others can use them for their assessment preparation or just to work on their Scrum Master skills.
+
+Our current product goal is to provide a question library for all to use. In future this might change - if you have good ideas let us know!
```

Comment:
```suggestion
Our current product goal is to provide a question library for all to use. In future, this might change - if you have good ideas, let us know!
```